clans/src/main/java/me/mykindos/betterpvp/clans/clans/listeners/ClanEventListener.java

```diff
@@ -389,9 +389,27 @@ public void onJoinClanEvent(final MemberJoinClanEvent event) {
                 return;
             }
 
+            boolean hasAdministratingClient = false;
+            for (ClanMember clanMember : clan.getMembers()) {
+                final Optional<Client> clientOpt = clientManager.search().online(UUID.fromString(clanMember.getUuid()));
+                if (clientOpt.isPresent()) {
+                    final Client online = clientOpt.get();
+                    if (online.isAdministrating()) {
+                        hasAdministratingClient = true;
+                        break;
+                    }
+                }
+            }
             if (clan.getSquadCount() >= this.maxClanMembers) {
-                UtilMessage.simpleMessage(player, "Clans", "<alt2>Clan " + clan.getName() + "</alt2> has too many members or allies");
-                return;
+                if (!hasAdministratingClient) {
+                    UtilMessage.simpleMessage(player, "Clans", "<alt2>Clan " + clan.getName() + "</alt2> has too many members or allies");
+                    return;
+                }
+                clientManager.sendMessageToRank("Clans",
+                        UtilMessage.deserialize("<yellow>%s</yellow> was able to join <yellow>%s</yellow> because there is a player in administrative mode in their clan",
+                                player.getName(), clan.getName()),
+                        Rank.HELPER);
+
             }
         } else {
             this.clientManager.sendMessageToRank("Clans", UtilMessage.deserialize("<yellow>%s<gray> force joined <yellow>%s", player.getName(), clan.getName()), Rank.HELPER);
```

Comment:
The invite data stores a gamer, we could probably check if the client who invited them is administrating instead of needing to do this

Reply:
It stores the inviter as the clan, not the gamer that invited them.